$NetBSD$

Enable NetBSD.
NetBSD's curses are good enough, so do not insist on ncurses.
Check for sys/statvfs.h.

--- configure.orig	2010-10-05 22:27:16.000000000 +0000
+++ configure
@@ -11069,10 +11069,10 @@ case $uname in
     WANT_KVM=yes
     WANT_DEVSTAT=yes
     ;;
-#  NetBSD*)
-#    WANT_KVM=yes
-#    WANT_OSSLIB=yes
-#    ;;
+  NetBSD*)
+    WANT_KVM=yes
+    WANT_OSSLIB=yes
+    ;;
 
   OpenBSD*)
     WANT_KVM=yes
@@ -11202,7 +11202,7 @@ else
 fi
 
 if test x$want_ncurses = xyes; then
-      conky_LIBS="$conky_LIBS -lncurses"
+      conky_LIBS="$conky_LIBS -lcurses"
 
 $as_echo "#define NCURSES 1" >>confdefs.h
 
@@ -16042,7 +16042,7 @@ fi
 fi
 
 for ac_header in signal.h unistd.h sys/utsname.h sys/stat.h linux/soundcard.h alsa/asoundlib.h dirent.h mcheck.h \
-      sys/statfs.h sys/param.h pthread.h semaphore.h assert.h errno.h time.h
+      sys/statfs.h sys/statvfs.h sys/param.h pthread.h semaphore.h assert.h errno.h time.h
 do :
   as_ac_Header=`$as_echo "ac_cv_header_$ac_header" | $as_tr_sh`
 ac_fn_c_check_header_mongrel "$LINENO" "$ac_header" "$as_ac_Header" "$ac_includes_default"
